### PR TITLE
Activity Log: move from "Stats > Activity" to it's own section

### PR DESCRIPTION
**Pull request by roccotripaldi**
This PR moved the activity log from the current location that is a tab under the activity log to the sidebar. 

:before:
screen_shot_2018-08-28_at_2_00_38_pm

:after:
screen_shot_2018-08-28_at_1_59_29_pm

Everything should still work like before. 

Related: p9rlnk-u5-p2

To test:
1. Run the branch locally, or on calypso.live
2. Ensure the 'Activity' submenu no longer appears on the stats page
3. Ensure that manually going to 'stats/activity' redirects to 'activity-log'
4. There should be an 'activity' link in the side bar
4. The activity log should be visible and function as before, but now living at '/activity-log' 
5. Guided tour for activity log should function as before

### CODE REVIEW

**Comment by enejb** [comment]
cc: @keoshi couple of questions about the design.

Does the icon make sense? For site that do not have rewind and will never user that feature I think something different might be more appropriate. The current icons is the one that is used on Mobile.

Does the positioning make sense? Should we put it lower down in int the sidebar?

**Comment by keoshi** [comment]
@enejb @roccotripaldi Whoa, that was quick! Position-wise, it makes total sense.

TL;DR: let's go with that icon for now unless we have a better suggestion.

I must say I'm unsure about that icon though. While it makes a lot of sense in the context of Rewind, it does not encapsulate what users should expect from “Activity” _right now_, because it doesn't any a direct correlation with the current UI.

Looking at available [Gridicons](http://automattic.github.io/gridicons/) we don't have a better one. `list-checkmark` sort of conveys the idea, but it's very bland and not future-proof.

And I guess the `history` icon we're using kind of hints at past events in an abstract way with the arrow and clock, and has enough visual differentiation in the sidebar, so let's go with that for now unless you have another suggestion. We can regroup all team members next week (after AFK time) and figure what something more relevant would look like.

I would use the same icon across all sites, to avoid confusion in a) documentation but also b) users that have both types of sites and see the icon changing in the sidebar for no apparent reason.

**Comment by alisterscott** [comment]
PR to update the e2e tests for activity log: https://github.com/Automattic/wp-e2e-tests/pull/1446

The e2e test PR should be merged straight after this PR is merged

**Comment by shaunandrews** [comment]
FWIW, I'm not a fan of this change. Not only is our sidebar already a "kitchen sink" of _stuff_, I also think there's going to be confusion between the words "Stats" and "Activity".

**Comment by roccotripaldi** [comment]
thanks Elio, fixed in:
https://github.com/Automattic/wp-calypso/pull/26941/commits/c859bd72699149a602640b8fc1ae04048579fa02

**Comment by roccotripaldi** [comment]
Nice catch @keoshi - fixed in https://github.com/Automattic/wp-calypso/pull/26941/commits/c9be3ac0d0e90a42c4f40a87ddc8e5c44d16e491
